docker-compose.yml

```diff
@@ -60,20 +60,21 @@ services:
         condition: service_healthy
     env_file: .env
     environment:
-      - RF_LOG_LEVEL=INFO
-      - TILE_SERVER_LOCATION=http://localhost:8080
+      - RF_LOG_LEVEL=DEBUG
+      - TILE_SERVER_LOCATION=
       - COURSIER_CACHE=/root/.coursier
     ports:
       - "9000:9000"
       - "9010:9010"
     volumes:
       - ./app-backend/:/opt/raster-foundry/app-backend/
       - ./scratch/:/opt/raster-foundry/scratch/
+      - ./data/:/opt/data/
```

Comment:
What is this volume mount used for?

Reply:
`./scripts/load-development-data` now includes downloading tiffs and placing it in this directory. This lets us test the tile and api servers while limiting network i/o to S3 by adding scenes with the `file://opt/data/<path-to-tiff>` prefix